File: app/src/main/java/com/yang/tally/AboutActivity.java
package com.yang.tally;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.view.View;

public class AboutActivity extends AppCompatActivity {

    private int test;
    private double a;
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_about);
    }

    public void onClick(View view) {
        finish();
    }
}

File: app/src/main/java/com/yang/tally/AboutActivity.kt
package com.yang.tally

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.view.View
import com.yang.tally.R

class AboutActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_about)
    }

    fun onClick(view: View?) {
        finish()
    }
}